### PR TITLE
HTML/CSS Max-Stupnitskyi

### PR DESCRIPTION
**Pull request by MaxStupnitskyi**
верстка інтерфейсу прогнозу погоди

### CODE REVIEW

File: homeworks/html-css/MaxStupnitskyi/index.html [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+
+		</div>
+
+		<div class="week">
+			
+			<div class="item">
+
+				<div class="day">Sunday</div>
+
+				<div class="icon"><img src="img/iconfinder_weather02_4102326.svg"></div>
```
Comment (maxkharena) [inline homeworks/html-css/MaxStupnitskyi/index.html]: alt for img tag is  required

File: homeworks/html-css/MaxStupnitskyi/index.html [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+	<title>Forecast</title>
+
+</head>
+
+<body>
+	
+	<main class="content">
+		
+		<h1 class="title">New York, NY</h1>
+
```
Comment (maxkharena) [inline homeworks/html-css/MaxStupnitskyi/index.html]: remove empty space between tags

Comment (maxkharena) [reply homeworks/html-css/MaxStupnitskyi/index.html]: From all HTML file

Comment (MaxStupnitskyi) [reply homeworks/html-css/MaxStupnitskyi/index.html]: Done

**Comment by MaxStupnitskyi** [comment]
> Please fix, according to my comments
Done

**Pull request merged**